src/bindings/python/flux/asyncio/__init__.py

```diff
@@ -0,0 +1 @@
+from .events import FluxEventLoop, loop, submit
```

Comment:
I think my preference is generally for relative imports - it's not a strong preference, but they look more succinct, are more portable (if I decide to move as a developer) and I don't know if this is a real concern or just an anxiety I have - but without relative, someone could add a different "flux" (or module name) to the PYTHONPATH to be found before wherever the main flux is, and then import from there instead of the expected location. Perhaps the relative import helps prevent that?

Reply:
This imports conversation can be followed up in the thread below (we currently have two).